changedetectionio/content_fetchers/puppeteer.py

```diff
@@ -8,6 +8,7 @@
 from changedetectionio.content_fetchers.base import Fetcher
 from changedetectionio.content_fetchers.exceptions import PageUnloadable, Non200ErrorCodeReceived, EmptyReply, BrowserFetchTimedOut, BrowserConnectError
 
+DEFAULT_USER_AGENT = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36"
```

Comment:
this should be set in the __init__.py of `content_fetchers` so all kinds can default to it

Reply:
and also browsersteps headers https://github.com/dgtlmoon/changedetection.io/issues/2197